packages/provider/src/transcription-model/v1/transcription-model-v1-call-options.ts

```diff
@@ -0,0 +1,35 @@
+import { JSONValue } from '../../json-value/json-value';
+
+export type TranscriptionModelV1CallOptions = {
+  /**
+Audio data to transcribe.
+     */
+  audio: Blob | File | Uint8Array | ArrayBuffer | Buffer | string;
```

Comment:
Let's chat about this, ideally it should be minimal e.g. `UInt8Array | string (base64)`